sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClient.java

```diff
@@ -512,14 +512,14 @@ public ConfigurationSetting deleteSetting(ConfigurationSetting setting) {
      *
      * <p>Retrieve all revisions of the setting that has the key "prodDBConnection".</p>
      *
-     * {@codesnippet com.azure.data.applicationconfig.configurationclient.listSettingRevisions#SettingSelector-Context}
+     * {@codesnippet com.azure.data.applicationconfig.configurationclient.listSettingRevisions#settingSelector-context}
      *
      * @param selector Optional. Used to filter configuration setting revisions from the service.
      * @param context Additional context that is passed through the Http pipeline during the service call.
-     * @return Revisions of the ConfigurationSetting
+     * @return {@link PagedIterable} of the {@link ConfigurationSetting}
```

Comment:
nit: This must be updated to read `{@link PagedIterable} of {@link ConfigurationSetting} revisions`